src/status_im2/contexts/onboarding/syncing/results/style.cljs

```diff
@@ -0,0 +1,28 @@
+(ns status-im2.contexts.onboarding.syncing.results.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def navigation-bar {:height 56})
```

Comment:
i've seen @alwx removed this

Reply:
@flexsurfer looks like in this case its still needed but it will be revised once the designs for these screens on syncing are completed. https://www.figma.com/file/V6nlpAWIf2e1XU8RJ9yQPe/Syncing-for-Mobile?node-id=675%3A179926&t=GxqZ9yWJ25csXbWq-1